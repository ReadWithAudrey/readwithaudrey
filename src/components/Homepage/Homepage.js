import React from 'react'
import HomepageStyled from './Homepage.style'
import { Link } from 'gatsby'
import { Button } from '../index'
import PropTypes from 'prop-types'
import bookpeople from '../../images/testrwalogo.png'

const Homepage = () => (
  <HomepageStyled>
    <div id="homepage-top" className="flex flex-column items-center justify-center">
      <h1 className="pink tc montserrat">Read Aloud Together</h1>
      <Button><Link to="/Form1/" className="white">Get started</Link></Button>
    </div>
    <img src={ bookpeople } alt="Cartoon of people reading a book"/>
  </HomepageStyled>
)

Homepage.propTypes = {
  siteTitle: PropTypes.string,
}

export default Homepage
