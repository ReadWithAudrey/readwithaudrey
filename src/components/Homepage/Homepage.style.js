import styled from "styled-components"

const HomepageStyled = styled.div.attrs({
  className: "bg-white flex flex-column items-center justify-around ph2 pv5 montserrat w-100",
})`
  height: 100vh;
`
export default HomepageStyled;