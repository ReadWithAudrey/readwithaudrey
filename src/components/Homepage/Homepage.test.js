import React from 'react'
import renderer from 'react-test-renderer'
import Homepage from './Homepage'

describe('Homepage component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<Homepage />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
