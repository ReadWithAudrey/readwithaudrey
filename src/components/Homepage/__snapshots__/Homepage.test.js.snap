// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage component snapshot test renders correctly 1`] = `
<div
  className="sc-bdVaJa bg-white flex flex-column items-center justify-between w-100 vh-100 c-padding fePDys"
>
  <div
    className="flex flex-column items-center justify-center"
    id="homepage-top"
  >
    <h1
      className="pink tc montserrat"
    >
      Read Aloud Together
    </h1>
    <Link
      className="white"
      to="/Form1/"
    >
      <button
        className="sc-gzVnrw bg-pink hover-bg-yellow white pa2 br2 w-50 h3 f3 montserrat bBtqGg"
      >
        Get started
      </button>
    </Link>
  </div>
  <img
    alt="Cartoon of people reading a book"
    src="test-file-stub"
  />
</div>
`;
